SendSignatureRequestMessage/function.json

```diff
@@ -4,7 +4,7 @@
       "name": "sendSignatureRequestMessage",
       "type": "queueTrigger",
       "direction": "in",
-      "queueName": "waiting-for-signature",
+      "queueName": "waiting-for-message",
```

Comment:
you may use the environment variable: `%QUEUE_STORAGE_NAME_FOR_WAITING_MESSAGE%`

Reply:
you are right, thanks!

Reply:
it is possible to directly use the environment variable with the syntax `%%`, so I entered `%WaitingMessageQueueName%` directly!